pkg/controller/awsfederatedaccountaccess/awsfederatedaccountaccess_controller.go

```diff
@@ -192,11 +194,24 @@ func (r *ReconcileAWSFederatedAccountAccess) Reconcile(request reconcile.Request
 		return reconcile.Result{}, err
 	}
 
+	configMap, err := controllerutils.GetOperatorConfigMap(r.client)
+	if err != nil {
+		log.Error(err, "failed retrieving configmap")
+	}
+	config.IsFedramp, err = controllerutils.IsFedramp(configMap)
```

Comment:
so, you're setting a global variable here, but update it on each reconcile, which seems like a dangerous pattern, can you think of a different way to solve this?
At least, I think you could hide that complexity inside the config package, so the global var is not set outside.

Reply:
i've moved checking for fedramp into operator startup

NB this will mean the operator will need to be restarted if the fedramp value in the configmap changes